net-libs/libwebsockets/metadata.xml

```diff
@@ -19,12 +19,10 @@
 		libwebsockets is a C library designed to make development of websockets clients and servers easier (possible).
 	</longdescription>
 	<use>
-		<flag name="ssl">Include SSL support (default <pkg>dev-libs/openssl</pkg>)</flag>
-		<flag name="zlib">Include zlib support</flag>
-		<flag name="static-libs">Build the static version of the library</flag>
+		<flag name="client">The client part of the library</flag>
```

Comment:
It means that you can use the library to build applications which are websocket clients, ie, they connect to a ws server as a client and can exchange information that way.

Without it enabled, you can only build applications that are websockets servers, and will be connected to by other websocket clients like web browsers.

With it enabled, you get another test app `libwebsockets-test-client`.  Otherwise it's just the library is a bit larger.